docs/source/asr/models.rst

```diff
@@ -59,6 +59,27 @@ obtain highly accurate audio transcripts while utilizing a non-autoregressive CT
 
 Citrinet models can be instantiated using the :class:`EncDecCTCModelBPE<nemo.collections.asr.models.EncDecCTCModelBPE>` class.
 
+
+.. _Conformer-CTC_model:
+
+Conformer-CTC
+-------------
+
+Conformer-CTC is a CTC-based variant of Conformer model introduced in :cite:`asr-models-gulati2020conformer`.
+Conformer-CTC used similar encoder as original Conformer but employs CTC loss and decoding instead of RNNT which makes it a non-autoregressive model.
+We also drops the LSTM decoder and instead use a linear decoder on the top of the encoder.
```

Comment:
This line should be removed. It's understood from the CTC decoding part above.

Reply:
For us, it may be obvious, but not very evident for others. Actually when I started, I used LSTM decoder on top of the Conformer, and still we have that decoder in nemo available.